src/core/Akka.API.Tests/CoreAPISpec.ApproveRemote.approved.txt

```diff
@@ -509,7 +510,10 @@ namespace Akka.Remote.Transport
         public Akka.Actor.Address LocalAddress { get; set; }
         public System.Threading.Tasks.TaskCompletionSource<Akka.Remote.Transport.IHandleEventListener> ReadHandlerSource { get; set; }
         public Akka.Actor.Address RemoteAddress { get; set; }
+        [System.ObsoleteAttribute("Use the method that states reasons to make sure disassociation reasons are logged" +
+            ".")]
         public abstract void Disassociate();
+        public void Disassociate(string reason, Akka.Event.ILoggingAdapter log) { }
```

Comment:
Added the ability to log explicit disassociation messages in `DEBUG` loglevel, in order to make troubleshooting connection issues easier. Consistent with JVM API.

Reply:
No - this new method is only called by Akka.Remote itself inside its remoting `/system` actors. Won't cause any problems.